core-library/src/main/java/org/silverpeas/core/security/session/SessionManagement.java

```diff
@@ -105,29 +90,17 @@ public interface SessionManagement {
    * by the user to Silverpeas, its current session must be validated. The validation updates also
    * useful information about the session like the timestamp of this access so that additional
    * features can be performed (for example, the timeout computation of the session).
-   *
    * @param context the context of the validation that contains at least the session key
    * @return information about the session identified by the specified key or null if no such
    * session exists.
    */
   SessionInfo validateSession(SessionValidationContext context);
 
-  /**
-   * Opens a new session for the specified user. This method is dedicated to open session off any
-   * WEB browsing context with HTTP servlets.
-   *
-   * @param user the user for which a session with Silverpeas has to be opened.
-   * @return a SessionInfo instance representing the current opened session with information about
-   * that session.
-   */
-  SessionInfo openSession(final User user);
```

Comment:
No it was previously defined when we started to write the first REST-based web services in order to open a virtual session for the user behind the request (to avoid to authenticate himself for each request to a REST resource).

Now, this isn't anymore the case. The session opened in Silverpeas is now fully based upon the HTTP session managed by the web container and hence it isn't anymore virtual.